src/code/qualification/getMolecularWeightForPath.m

```diff
@@ -20,6 +20,11 @@
 
 jj = cellfun(@(x) contains([outputPath '|'],x),strcat('|',compounds,'|'));
 
+if sum(jj)~=1
+    disp(jj);
+    jj=1; 
+end
+

```

Comment:
can you explain what happens here in the lines 23-26?

Reply:
These lines were for me to debug the issue with getMolecularWeightForPath: for some Paths the cellfun seems unable to find the compound and provides an empty output leading to issue #139. 
I have forgotten to replace the lines with a warning or error message.